cli/cmd/install.go

```diff
@@ -142,9 +115,8 @@ type (
 		recordedFlags []*pb.Install_Flag
 
 		// function pointers that can be overridden for tests
-		generateUUID       func() string
-		generateWebhookTLS func(webhook string) (*tlsValues, error)
```

Comment:
During install, the webhook TLS secrets will be generated by Helm functions. During an upgrade, these secrets will be preserved.